components/SearchFilter.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+interface SearchFilterProps {
```

Comment:
small nitpick just use `type` instead of interface for now

Reply:
sounds good. I guess habit forced me to do interface, but question, is there specific times you would you one over the other? Is the logic behind interfaces more when other React components are part of its props ?